🐙 source-trustpilot: run up-to-date pipeline [2025-02-01]

airbyteio opened the pull request:
# Update source-trustpilot

This PR was autogenerated by running `airbyte-ci connectors --name=source-trustpilot up_to_date --pull`

We've set the `auto-merge` label on it, so it will be automatically merged if the CI pipelines pass.
If you don't want to merge it automatically, please remove the `auto-merge` label.
Please reach out to the Airbyte Connector Tooling team if you have any questions or concerns.


## Operations

- Upgrade the base image to the latest version in metadata.yaml: Successful

- PATCH bump source-trustpilot version to 0.3.10: Successful

- Build source-trustpilot docker image for platform(s) linux/amd64, linux/arm64: Successful

- Get dependency updates: Successful

- Create or update pull request on Airbyte repository: Successful

- Add changelog entry: Successful




## Dependency updates

We use [`syft`](https://github.com/anchore/syft) to generate a SBOM for the latest connector version and the one from the PR.
It allows us to spot the dependencies that have been updated at all levels and for all types of dependencies (system, python, java etc.).
Here are the dependencies that have been updated compared to `airbyte/source-trustpilot:latest`.
Keep in mind that `:latest` does not always match the connector code on the main branch. 
It is the latest released connector image when the head commit of this branch was created.

| Type | Name | State | Previous Version | New Version |
|------|------|-------|-------------|------------------|
| python | airbyte-cdk | updated | 6.27.0 | **6.31.1** |
| python | certifi | updated | 2024.12.14 | **2024.8.30** |
| python | charset-normalizer | updated | 3.4.0 | **3.4.1** |
| python | urllib3 | updated | 2.2.3 | **2.3.0** |
| python | whenever | added | not present | **0.6.16** |
| python | pendulum | removed | 2.1.2 | **not present** |
| python | pytzdata | removed | 2020.1 | **not present** |

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/airbytehq/airbyte?pullRequest=53104) <br/>All committers have signed the CLA.

airbyteio committed:
Upgrade the base image to the latest version in metadata.yaml
PATCH bump source-trustpilot version to 0.3.10
Build source-trustpilot docker image for platform(s) linux/amd64, linux/arm64
Get dependency updates
Create or update pull request on Airbyte repository
Add changelog entry
